architecture-building-systems/CityEnergyAnalyst
# Add multiple network cal procedure

This pull request modifies the `thermal_network_matrix.py` to iterate through multiple networks within a district.

To test:
- sync 'cea-reference-case'
- there should be four folders (All_building, RTP, SBS, N3_1) under this directory `\reference-case-ecocampus\baseline\inputs\networks\DC'
- run 'thermal_network_matrix.py' with the scenario 'reference-case-ecocampus`
- the script should execute the DC network calculation for all four networks
